espresso-server/app/src/test/java/io/appium/espressoserver/test/helpers/w3c/TickTest.java

```diff
@@ -208,15 +208,15 @@ public void shouldDispatchPointerMoveEvents() throws AppiumException, Interrupte
             @Override
             public void pointerMove(String sourceIdCalled,
                                     PointerType pointerType,
-                                    float currentX, float currentY,
-                                    float x, float y,
+                                    Float currentX, Float currentY,
+                                    Float x, Float y,
                                     Set<Integer> buttons,
                                     KeyInputState globalKeyInputState) {
                 assertEquals(pointerType, TOUCH);
-                assertEquals(currentX, 5L);
-                assertEquals(currentY, 6L);
-                assertEquals(x, 10L);
-                assertEquals(y, 20L);
+                assertEquals(currentX, 5, Math.ulp(1.0));
```

Comment:
so how about moving `Math.ulp(1.0)` to a constant?

Reply:
I refactored it to use a helper method